<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Julia Kr√∂ger">
<meta name="dcterms.date" content="2025-04-16">

<title>Literature Collection</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="Literature_files/libs/clipboard/clipboard.min.js"></script>
<script src="Literature_files/libs/quarto-html/quarto.js"></script>
<script src="Literature_files/libs/quarto-html/popper.min.js"></script>
<script src="Literature_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="Literature_files/libs/quarto-html/anchor.min.js"></script>
<link href="Literature_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="Literature_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="Literature_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="Literature_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="Literature_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#download-bibliography" id="toc-download-bibliography" class="nav-link active" data-scroll-target="#download-bibliography">Download Bibliography</a></li>
  <li><a href="#microbal-production-of-sphingolipids" id="toc-microbal-production-of-sphingolipids" class="nav-link" data-scroll-target="#microbal-production-of-sphingolipids">‚ö´Ô∏èMicrobal Production of Sphingolipids</a>
  <ul>
  <li><a href="#tabs-production" id="toc-tabs-production" class="nav-link" data-scroll-target="#tabs-production">‚óæÔ∏èTABS Production</a>
  <ul class="collapse">
  <li><a href="#zhang-et-al.-2024" id="toc-zhang-et-al.-2024" class="nav-link" data-scroll-target="#zhang-et-al.-2024">Zhang et al.&nbsp;(2024)</a></li>
  <li><a href="#han-et-al.-2020" id="toc-han-et-al.-2020" class="nav-link" data-scroll-target="#han-et-al.-2020">Han et al.&nbsp;(2020)</a></li>
  <li><a href="#yoo-et-al.-2023" id="toc-yoo-et-al.-2023" class="nav-link" data-scroll-target="#yoo-et-al.-2023">Yoo et al.&nbsp;(2023)</a></li>
  <li><a href="#eun-and-lee-2023" id="toc-eun-and-lee-2023" class="nav-link" data-scroll-target="#eun-and-lee-2023">Eun and Lee (2023)</a></li>
  <li><a href="#yoo-et-al.-2023-1" id="toc-yoo-et-al.-2023-1" class="nav-link" data-scroll-target="#yoo-et-al.-2023-1">Yoo et al.&nbsp;(2023)</a></li>
  </ul></li>
  <li><a href="#other-sphingolipids" id="toc-other-sphingolipids" class="nav-link" data-scroll-target="#other-sphingolipids">‚óæÔ∏èOther Sphingolipids</a>
  <ul class="collapse">
  <li><a href="#murakami-et-al.-2015" id="toc-murakami-et-al.-2015" class="nav-link" data-scroll-target="#murakami-et-al.-2015">Murakami et al.&nbsp;(2015)</a></li>
  <li><a href="#shin-et-al.-2024" id="toc-shin-et-al.-2024" class="nav-link" data-scroll-target="#shin-et-al.-2024">Shin et al.&nbsp;(2024)</a></li>
  <li><a href="#b√∂rgel-et-al.-2012" id="toc-b√∂rgel-et-al.-2012" class="nav-link" data-scroll-target="#b√∂rgel-et-al.-2012">B√∂rgel et al.&nbsp;(2012)</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#Œ¥lag1Œ¥lac1-literature" id="toc-Œ¥lag1Œ¥lac1-literature" class="nav-link" data-scroll-target="#Œ¥lag1Œ¥lac1-literature">üü£Œîlag1Œîlac1 Literature</a>
  <ul>
  <li><a href="#epstein-et-al.-2012" id="toc-epstein-et-al.-2012" class="nav-link" data-scroll-target="#epstein-et-al.-2012">Epstein et al.&nbsp;(2012)</a></li>
  <li><a href="#cerantola-et-al.-2007" id="toc-cerantola-et-al.-2007" class="nav-link" data-scroll-target="#cerantola-et-al.-2007">Cerantola et al.&nbsp;(2007)</a></li>
  <li><a href="#guillas-et-al.-2001" id="toc-guillas-et-al.-2001" class="nav-link" data-scroll-target="#guillas-et-al.-2001">Guillas et al.&nbsp;(2001)</a></li>
  <li><a href="#kol-et-al.-2025" id="toc-kol-et-al.-2025" class="nav-link" data-scroll-target="#kol-et-al.-2025">Kol et al.&nbsp;(2025)</a></li>
  <li><a href="#ternes-et-al.-2009" id="toc-ternes-et-al.-2009" class="nav-link" data-scroll-target="#ternes-et-al.-2009">Ternes et al.&nbsp;(2009)</a></li>
  <li><a href="#vionnet-et-al.-2011" id="toc-vionnet-et-al.-2011" class="nav-link" data-scroll-target="#vionnet-et-al.-2011">Vionnet et al.&nbsp;(2011)</a></li>
  <li><a href="#cerantola-et-al.-2007-1" id="toc-cerantola-et-al.-2007-1" class="nav-link" data-scroll-target="#cerantola-et-al.-2007-1">Cerantola et al.&nbsp;(2007)</a></li>
  <li><a href="#menuz-et-al.-2009" id="toc-menuz-et-al.-2009" class="nav-link" data-scroll-target="#menuz-et-al.-2009">Menuz et al.&nbsp;(2009)</a></li>
  </ul></li>
  <li><a href="#sphingomyelin-synthase-expression-in-yeast" id="toc-sphingomyelin-synthase-expression-in-yeast" class="nav-link" data-scroll-target="#sphingomyelin-synthase-expression-in-yeast">üî¥Sphingomyelin Synthase Expression in Yeast</a>
  <ul>
  <li><a href="#ternes-et-al.-2009-1" id="toc-ternes-et-al.-2009-1" class="nav-link" data-scroll-target="#ternes-et-al.-2009-1">Ternes et al.&nbsp;(2009)</a></li>
  <li><a href="#yang-et-al.-2007" id="toc-yang-et-al.-2007" class="nav-link" data-scroll-target="#yang-et-al.-2007">Yang et al.&nbsp;(2007)</a></li>
  </ul></li>
  <li><a href="#chemoenzymatic-ganglioside-synthesis" id="toc-chemoenzymatic-ganglioside-synthesis" class="nav-link" data-scroll-target="#chemoenzymatic-ganglioside-synthesis">üü†Chemoenzymatic Ganglioside Synthesis</a>
  <ul>
  <li><a href="#jin-et-al.-2024" id="toc-jin-et-al.-2024" class="nav-link" data-scroll-target="#jin-et-al.-2024">Jin et al.&nbsp;(2024)</a></li>
  <li><a href="#han-et-al.-2015" id="toc-han-et-al.-2015" class="nav-link" data-scroll-target="#han-et-al.-2015">Han et al.&nbsp;(2015)</a></li>
  <li><a href="#huang-et-al.-2015" id="toc-huang-et-al.-2015" class="nav-link" data-scroll-target="#huang-et-al.-2015">Huang et al.&nbsp;(2015)</a></li>
  </ul></li>
  </ul>
</nav>
</div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Literature Collection</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Julia Kr√∂ger </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">April 16, 2025</p>
    </div>
  </div>
  
    
  </div>
  


</header>


<div class="cell">
<style>
p {
  text-align: justify;
}
body {
  padding-top: 80px;
}
</style>
</div>
<hr>
<section id="download-bibliography" class="level4">
<h4 class="anchored" data-anchor-id="download-bibliography">Download Bibliography</h4>
<ul>
<li><a href="literature.bib">BibTeX-Version</a><br>
</li>
<li><a href="zotero-version.rdf">Zotero-Version (.rdf)</a></li>
</ul>
<hr>
<hr>
</section>
<section id="microbal-production-of-sphingolipids" class="level2">
<h2 class="anchored" data-anchor-id="microbal-production-of-sphingolipids">‚ö´Ô∏èMicrobal Production of Sphingolipids</h2>
<blockquote class="blockquote">
<p>These studies aim to produce some sort of sphingolipid or ceramide using yeast.</p>
</blockquote>
<section id="tabs-production" class="level3">
<h3 class="anchored" data-anchor-id="tabs-production">‚óæÔ∏èTABS Production</h3>
<details>
<summary>
<strong>Show Publications</strong>
</summary>
<section id="zhang-et-al.-2024" class="level4">
<h4 class="anchored" data-anchor-id="zhang-et-al.-2024">Zhang et al.&nbsp;(2024)</h4>
<section id="advances-in-the-biosynthesis-of-tetraacetyl-phytosphingosine-a-key-substrate-of-ceramides" class="level5">
<h5 class="anchored" data-anchor-id="advances-in-the-biosynthesis-of-tetraacetyl-phytosphingosine-a-key-substrate-of-ceramides">Advances in the biosynthesis of tetraacetyl phytosphingosine, a key substrate of ceramides</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>Advances in the biosynthesis of tetraacetyl phytosphingosine, a key substrate of ceramides.</strong><br>
<em>Xin Zhang, Xiaochen Zhang, Lu Lin , Kaifeng Wang, Xiao-Jun Ji</em></p>
</blockquote>
<p>Ceramides, formed by the dehydration of long-chain fatty acids with phytosphingosine and its derivatives, are widely used in skincare, cosmetics, and pharmaceuticals. Due to the exceedingly low concentration of phytosphingosine in plant seeds, relying on the extraction method is highly challenging. Currently, the primary method for obtaining phytosphingosine is the deacetylation of tetraacetyl phytosphingosine (TAPS) derived from fermentation. Wickerhamomyces ciferrii, an unconventional yeast from the pods of Dipteryx odorata, is the only known microorganism capable of naturally secreting TAPS, which is of great industrial value. In recent years, research and applications focused on modifying W. ciferrii for TAPS overproduction have increased rapidly. This review first describes the discovery history, applications, microbial synthesis pathway of TAPS. Research progress in using haploid breeding, mutagenesis breeding, and metabolic engineering to improve TAPS production is then summarized. In addition, the future prospects of TAPS production using the W. ciferrii platform are discussed in light of the current progress, challenges, and trends in this field. Finally, guidelines for future researches are also emphasized.</p>
<p><a href="https://doi.org/10.1016/j.synbio.2024.07.005">10.1016/j.synbio.2024.07.005</a></p>
</details>
<hr>
</section>
</section>
<section id="han-et-al.-2020" class="level4">
<h4 class="anchored" data-anchor-id="han-et-al.-2020">Han et al.&nbsp;(2020)</h4>
<section id="engineering-yarrowia-lipolytica-for-de-novo-production-of-tetraacetyl-phytosphingosine" class="level5">
<h5 class="anchored" data-anchor-id="engineering-yarrowia-lipolytica-for-de-novo-production-of-tetraacetyl-phytosphingosine">Engineering <em>Yarrowia lipolytica</em> for de novo production of tetraacetyl phytosphingosine</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>Engineering <em>Yarrowia lipolytica</em> for de novo production of tetraacetyl phytosphingosine.</strong><br>
<em>Changpyo Han, Minjeong Jang, Min Ju Kim, Man-Ho Han, Kyeong-Ryoon Lee, Ji-Sook Hahn and Jungoh Ahn</em></p>
</blockquote>
<p>Aims: To genetically engineer the oleaginous yeast Yarrowia lipolytica for de novo production of tetraacetylphytosphingosine (TAPS), a precursor of phytosphingosine, and optimization of fermentation conditions for high yield. Methods and Results: We successfully constructed a TAPS-producing Y. lipolytica CE3 strain by co-expression of Wickerhamomyces ciferrii-derived acetyl transferases, Sli1p and Atf2p. Next, we optimized several environmental factors including temperature, initial pH and C/N ratio for TAPS production in a shake culture. Deletion of LCB4 in CE3 strain increased the volumetric TAPS titre and cell-specific yield to 142 1 10 7 mgTAPS l 1 and 3 08 0 11 mgTAPS gDCW 1, respectively, in a shake flask culture incubated for 120 h at 28¬∞C with glycerol as the carbon source. Finally, we developed a 5-l fed-batch process with NaOH-mediated pH control and olive oil as a carbon source, exhibiting 650 24 mgTAPS l 1 of TAPS production within 56 h of the fermentation.</p>
<p><a href="https://doi.org/10.1111/jam.14931">10.1111/jam.14931</a></p>
</details>
<hr>
</section>
</section>
<section id="yoo-et-al.-2023" class="level4">
<h4 class="anchored" data-anchor-id="yoo-et-al.-2023">Yoo et al.&nbsp;(2023)</h4>
<section id="wickerhamomyces-ciferrii-auxotroph-and-expression-vector-for-improved-production-of-tetraacetyl-phytosphingosine" class="level5">
<h5 class="anchored" data-anchor-id="wickerhamomyces-ciferrii-auxotroph-and-expression-vector-for-improved-production-of-tetraacetyl-phytosphingosine"><em>Wickerhamomyces ciferrii</em> Auxotroph and Expression Vector for Improved Production of Tetraacetyl Phytosphingosine</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong><em>Wickerhamomyces ciferrii</em> Auxotroph and Expression Vector for Improved Production of Tetraacetyl Phytosphingosine</strong> <em>Seok Woo Yoo, Minsun Kim, Boo-Su Park, Jihee Yoon, and Min-Kyu Oh</em></p>
</blockquote>
<p>Tetraacetyl phytosphingosine (TAPS) is a precursor that has antibacterial and wound-healing properties and is used in the synthesis of ceramides. Wickerhamomyces ciferrii is a non-conventional yeast that has the ability to produce and secrete TAPS extracellularly. However, the genetic engineering tools available for this yeast species are limited. In this study, an auxotrophic strain Y94 of W. ciferrii was developed through random mutation and screening under 1.5 g/L 5-fluoroorotic acid. Additionally, an expression vector was constructed which harbors an uracil selection marker and CEN/ARS origin of replication. Using requisite mutations and expression vectors, we successfully constructed an engineered strain that overexpressed LCB1, LCB2, and SYR2, with the deletion of LCB4, resulting in the highest reported TAPS titer to date (20 g/L) through high cell density fermentation. This study demonstrates the potential of W. ciferrii for TAPS production and highlights the significance of an efficient genetic engineering toolbox in enhancing yields.</p>
<p><a href="https://doi.org/10.1007/s12257-023-0128-y">10.1007/s12257-023-0128-y</a></p>
</details>
<hr>
</section>
</section>
<section id="eun-and-lee-2023" class="level4">
<h4 class="anchored" data-anchor-id="eun-and-lee-2023">Eun and Lee (2023)</h4>
<section id="investigation-of-the-effects-of-culture-conditions-on-cell-growth-and-tetraacetylphytosphingosine-production-by-mutant-wickerhamomyces-ciferrii" class="level5">
<h5 class="anchored" data-anchor-id="investigation-of-the-effects-of-culture-conditions-on-cell-growth-and-tetraacetylphytosphingosine-production-by-mutant-wickerhamomyces-ciferrii">Investigation of the effects of culture conditions on cell growth and tetraacetylphytosphingosine production by mutant <em>Wickerhamomyces ciferrii</em></h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>Investigation of the effects of culture conditions on cell growth and tetraacetylphytosphingosine production by mutant <em>Wickerhamomyces ciferrii</em></strong><br>
<em>Sung Wook Eun , Pyung Cheon Lee</em></p>
</blockquote>
<p>Wickerhamomyces ciferrii is a non-conventional yeast strain characterized by the production of large amounts of tetraacetylphytosphingosine (TAPS). TAPS is an industrially valuable substance that can be converted into bioactive ceramide through deacetylation. TAPS-overproducing W. ciferrii is typically cultured in complex media, but there have been no systematic studies on the development of a defined medium for TAPS production. In this study, the composition of a defined medium for W. ciferrii strain was systematically investigated to increase the TAPS production. Among the defined medium components tested, Zn2+ significantly influenced TAPS production. The modified defined medium devoid of ZnSO4, with 40 g/L glycerol as a carbon source, increased TAPS production by 35% (1.16 ¬± 0.04 g/L, 0.039 g-TAPS/OD600) compared to the control defined medium (0.8 ¬± 0.01 g/L, 0.022 g-TAPS/OD600) in flask cultures. In batch bioreactor fermentation using the modified defined medium (40 g/L glycerol), 1.85 ¬± 0.1 g/L of TAPS (0.046 g-TAPS/g-glycerol, 0.044 g-TAPS/OD600) was achieved, which was 28% higher than 1.45 ¬± 0.02 g/L of TAPS (0.036 g-TAPS/g-glycerol, 0.027 g-TAPS/OD600) with the complex medium. Therefore, the modified defined medium developed in this study has the potential to serve as a base medium for large- scale TAPS production and for physiological and metabolic engineering studies.</p>
<p><a href="https://doi.org/10.1016/j.procbio.2023.04.017">10.1016/j.procbio.2023.04.017</a></p>
</details>
<hr>
</section>
</section>
<section id="yoo-et-al.-2023-1" class="level4">
<h4 class="anchored" data-anchor-id="yoo-et-al.-2023-1">Yoo et al.&nbsp;(2023)</h4>
<section id="wickerhamomyces-ciferrii-auxotroph-and-expression-vector-for-improved-production-of-tetraacetyl-phytosphingosine-1" class="level5">
<h5 class="anchored" data-anchor-id="wickerhamomyces-ciferrii-auxotroph-and-expression-vector-for-improved-production-of-tetraacetyl-phytosphingosine-1"><em>Wickerhamomyces ciferrii</em> Auxotroph and Expression Vector for Improved Production of Tetraacetyl Phytosphingosine</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p>** <em>Wickerhamomyces ciferrii</em> Auxotroph and Expression Vector for Improved Production of Tetraacetyl Phytosphingosine.**<br>
<em>Seok Woo Yoo, Minsun Kim, Boo-Su Park, Jihee Yoon, and Min-Kyu Oh</em></p>
</blockquote>
<p>Tetraacetyl phytosphingosine (TAPS) is a precursor that has antibacterial and wound-healing properties and is used in the synthesis of ceramides. Wickerhamomyces ciferrii is a non-conventional yeast that has the ability to produce and secrete TAPS extracellularly. However, the genetic engineering tools available for this yeast species are limited. In this study, an auxotrophic strain Y94 of W. ciferrii was developed through random mutation and screening under 1.5 g/L 5-fluoroorotic acid. Additionally, an expression vector was constructed which harbors an uracil selection marker and CEN/ARS origin of replication. Using requisite mutations and expression vectors, we successfully constructed an engineered strain that overexpressed LCB1, LCB2, and SYR2, with the deletion of LCB4, resulting in the highest reported TAPS titer to date (20 g/L) through high cell density fermentation. This study demonstrates the potential of W. ciferrii for TAPS production and highlights the significance of an e</p>
<p><a href="https://doi.org/10.1007%2Fs12257-023-0128-y">10.1007%2Fs12257-023-0128-y</a></p>
</details>
<hr>
</section>
</section>
</details></section>
<section id="other-sphingolipids" class="level3">
<h3 class="anchored" data-anchor-id="other-sphingolipids">‚óæÔ∏èOther Sphingolipids</h3>
<details>
<summary>
<strong>Show Publications</strong>
</summary>
<section id="murakami-et-al.-2015" class="level4">
<h4 class="anchored" data-anchor-id="murakami-et-al.-2015">Murakami et al.&nbsp;(2015)</h4>
<section id="producing-human-ceramide-ns-by-metabolic-engineering-using-yeast-saccharomyces-cerevisiae" class="level5">
<h5 class="anchored" data-anchor-id="producing-human-ceramide-ns-by-metabolic-engineering-using-yeast-saccharomyces-cerevisiae">Producing human ceramide-NS by metabolic engineering using yeast <em>Saccharomyces cerevisiae</em></h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>Producing human ceramide-NS by metabolic engineering using yeast <em>Saccharomyces cerevisiae</em>.</strong><br>
<em>Suguru Murakami, Toshi Shimamoto, Hideaki Nagano, Masahiro Tsuruno, Hiroaki Okuhara, Haruyo Hatanaka, Hiromasa Tojo, Yukiko Kodama2 &amp; Kouichi Funato</em></p>
</blockquote>
<p>Ceramide is one of the most important intercellular components responsible for the barrier and moisture retention functions of the skin. Because of the risks involved with using products of animal origin and the low productivity of plants, the availability of ceramides is currently limited. In this study, we successfully developed a system that produces sphingosine-containing human ceramide-NS in the yeast Saccharomyces cerevisiae by eliminating the genes for yeast sphingolipid hydroxylases (encoded by SUR2 and SCS7) and introducing the gene for a human sphingolipid desaturase (encoded by DES1). The inactivation of the ceramidase gene YDC1, overexpression of the inositol phosphosphingolipid phospholipase C gene ISC1, and endoplasmic reticulum localization of the DES1 gene product resulted in enhanced production of ceramide-NS. The engineered yeast strains can serve as hosts not only for providing a sustainable source of ceramide-NS but also for developing further systems to produce sphingosine-containing sphingolipids.</p>
<p><a href="https://doi.org/10.1038/srep16319">10.1038/srep16319</a></p>
</details>
<hr>
</section>
</section>
<section id="shin-et-al.-2024" class="level4">
<h4 class="anchored" data-anchor-id="shin-et-al.-2024">Shin et al.&nbsp;(2024)</h4>
<section id="elucidation-and-engineering-of-sphingolipid-biosynthesis-pathway-in-yarrowia-lipolytica-for-enhanced-production-of-human-type-sphingoid-bases-and-glucosylceramides" class="level5">
<h5 class="anchored" data-anchor-id="elucidation-and-engineering-of-sphingolipid-biosynthesis-pathway-in-yarrowia-lipolytica-for-enhanced-production-of-human-type-sphingoid-bases-and-glucosylceramides">Elucidation and engineering of Sphingolipid biosynthesis pathway in <em>Yarrowia lipolytica</em> for enhanced production of human-type sphingoid bases and glucosylceramides</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>Elucidation and engineering of Sphingolipid biosynthesis pathway in <em>Yarrowia lipolytica</em> for enhanced production of human-type sphingoid bases and glucosylceramides.</strong><br>
<em>Seo Hyeon Shin, Hye Yun Moon, Hae Eun Park, Gi Jeong Nam, Ju Hye Baek, Che Ok Jeon a, Hyunwook Jung, Myeong Seok Cha, Sol Choi, Jeong Jun Han, Chen Yuan Hou, Chang Seo Park, Hyun Ah Kango</em></p>
</blockquote>
<p>Sphingolipids are vital membrane components in in mammalian cells, plants, and various microbes. We aimed to explore and exploit the sphingolipid biosynthesis pathways in an oleaginous and dimorphic yeast Yarrowia lipolytica by constructing and characterizing mutant strains with specific gene deletions and integrating exogenous genes to enhance the production of long-chain bases (LCBs) and glucosylceramides (GlcCers). To block the fungal/plant-specific phytosphingosine (PHS) pathway, we deleted the SUR2 gene encoding a sphinganine C4hydroxylase, resulting in a remarkably elevated secretory production of dihydrosphingosine (DHS) and sphingosine (So) without acetylation. The Y. lipolytica SUR2 deletion (Ylsur2Œî) strain displayed retarded growth, increased pseudohyphal formation and stress sensitivity, along with the altered profiles of inositolphosphatecontaining ceramides, GlcCers, and sterols. The subsequent disruption of the SLD1 gene, encoding a fungal/ plant-specific Œî8 sphingolipid desaturase, restored filamentous growth in the Ylsur2Œî strain to a yeast-type form and further increased the production of human-type GlcCers. Additional introduction of mouse alkaline ceramidase 1 (maCER1) into the Ylsur2Œîsld1Œî double mutants considerably increased DHS and So production while decreasing GlcCers. The production yields of LCBs from the Ylsur2Œîsld1Œî/maCER1 strain increased in proportion to the C/N ratio in the N-source optimized medium, leading to production of 1.4 g/L non-acetylated DHS at the 5 L fed-batch fermentation with glucose feeding. This study highlights the feasibility of using the engineered Y. lipolytica strains as a cell factory for valuable sphingolipid derivatives for pharmaceuticals, cosmeceuticals, and nutraceuticals.</p>
<p><a href="https://doi.org/10.1016/j.ymben.2024.11.013">10.1016/j.ymben.2024.11.013</a></p>
</details>
<hr>
</section>
</section>
<section id="b√∂rgel-et-al.-2012" class="level4">
<h4 class="anchored" data-anchor-id="b√∂rgel-et-al.-2012">B√∂rgel et al.&nbsp;(2012)</h4>
<section id="metabolic-engineering-of-the-non-conventional-yeast-pichia-ciferrii-for-production-of-rare-sphingoid-bases" class="level5">
<h5 class="anchored" data-anchor-id="metabolic-engineering-of-the-non-conventional-yeast-pichia-ciferrii-for-production-of-rare-sphingoid-bases">Metabolic engineering of the non-conventional yeast Pichia ciferrii for production of rare sphingoid bases</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>Metabolic engineering of the non-conventional yeast Pichia ciferrii for production of rare sphingoid bases.</strong><br>
<em>Daniel Borgel, Marco van den Berg, Thomas Huller, Heiko Andrea, Gerhard Liebisch, Eckhard Boles, Christoph Schorsch, Ruud van der Pol, Anne Arink, Ilco Boogers, Rob van der Hoeven, Kees Korevaar, Mike Farwick, Tim Kohler,3, Steffen Schaffer</em></p>
</blockquote>
<p>The study describes the identification of sphingolipid biosynthesis genes in the non-conventional yeast Pichia ciferrii, the development of tools for its genetic modification as well as their application for metabolic engineering of P. ciferrii with the goal to generate strains capable of producing the rare sphingoid bases sphinganine and sphingosine. Several canonical genes encoding ceramide synthase (encoded by PcLAG1 and PcLAF1), alkaline ceramidase (PcYXC1) and sphingolipid C-4-hydroxylase (PcSYR2), as well as structural genes for dihydroceramide D4-desaturase (PcDES1) and sphingolipid D8-desaturase (PcSLD1) were identified, indicating that P. ciferrii would be capable of synthesizing desaturated sphingoid bases, a property not ubiquitously found in yeasts. In order to convert the phytosphingosine-producing P. ciferrii wildtype into a strain capable of producing predominantly sphinganine, Syringomycin E-resistant mutants were isolated. A stable mutant almost exclusively producing high levels of acetylated sphinganine was obtained and used as the base strain for further metabolic engineering. A metabolic pathway required for the three-step conversion of sphinganine to sphingosine was implemented in the sphinganine producing P. ciferrii strain and subsequently enhanced by screening for the appropriate heterologous enzymes, improvement of gene expression and codon optimization. These combined efforts led to a strain capable of producing 240 mg L 1 triacetyl sphingosine in shake flask, with tri- and diacetyl sphinganine being the main by-products. Lab-scale fermentation of this strain resulted in production of up to 890 mg kg 1 triacetyl sphingosine. A third by-product was unequivocally identified as triacetyl sphingadienine. It could be shown that inactivation of the SLD1 gene in P. ciferrii efficiently suppresses triacetyl sphingadienine formation. Further improvement of the described P. ciferrii strains will enable a biotechnological route to produce sphinganine and sphingosine for cosmetic and pharmaceutical applications.</p>
<p><a href="http://dx.doi.org/10.1016/j.ymben.2012.03.003">10.1016/j.ymben.2012.03.003</a></p>
</details>
<hr>
</section>
</section>
</details></section>
</section>
<section id="Œ¥lag1Œ¥lac1-literature" class="level2">
<h2 class="anchored" data-anchor-id="Œ¥lag1Œ¥lac1-literature">üü£Œîlag1Œîlac1 Literature</h2>
<blockquote class="blockquote">
<p>These studies relate to investigating sphingolipid metabolism through yeast containing the ŒîLAG1ŒîLAC1 deletion, leading to loss of acyl-CoA-dependent ceramide biosynthesis.</p>
</blockquote>
<details>
<summary>
<strong>Show Publications</strong>
</summary>
<section id="epstein-et-al.-2012" class="level4">
<h4 class="anchored" data-anchor-id="epstein-et-al.-2012">Epstein et al.&nbsp;(2012)</h4>
<section id="an-essential-function-of-sphingolipids-in-yeast-cell-division" class="level5">
<h5 class="anchored" data-anchor-id="an-essential-function-of-sphingolipids-in-yeast-cell-division">An essential function of sphingolipids in yeast cell division</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>An essential function of sphingolipids in yeast cell division.</strong><br>
<em>Sharon Epstein, Guillaume A. Castillon, Yongmei Qin and Howard Riezman</em></p>
</blockquote>
<p>Ceramides are bioactive lipids and precursors to sphingolipids. They have been shown to take part in a wide variety of different physiological processes in eukaryotic organisms and are thought to be toxic at high concentrations. Ceramide is synthesized by condensation of the sphingoid base sphinganine and a fatty acyl CoA by ceramide synthases, a family of enzymes that differ in their specificity for the length of the acyl CoA substrate. <span style="background-color: #f8bbd0"> We have engineered a yeast strain where the endogenous ceramide synthase has been replaced by one of the putative enzymes from cotton. As a result, the yeast strain produces C18 rather than C26 ceramides showing that the cotton protein is a bona fide ceramide synthase with specificity towards C18 acyl CoA. Strikingly, the accumulation of C18 ceramide is not toxic in Saccharomyces cerevisiae.</span> This allows survival of the yeast after deletion of the normally essential AUR1 (inositol phosphorylceramide synthase) gene permitting us to address the essential roles of sphingolipids. Deletion of AUR1 allows cell growth, but leads to a defect in cytokinesis, which takes twice as long as in wild-type strains. Nuclear division and recruitment of septins is apparently not affected, but cytokinesis is delayed and cell separation is incomplete.</p>
<p><a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-2958.2012.08087.x">doi/10.1111/j.1365-2958.2012.08087.x</a></p>
</details>
<hr>
</section>
</section>
<section id="cerantola-et-al.-2007" class="level4">
<h4 class="anchored" data-anchor-id="cerantola-et-al.-2007">Cerantola et al.&nbsp;(2007)</h4>
<section id="sphingolipid-synthesis-is-required-for-heat-stress-resistance-in-yeast" class="level5">
<h5 class="anchored" data-anchor-id="sphingolipid-synthesis-is-required-for-heat-stress-resistance-in-yeast">Sphingolipid synthesis is required for heat stress resistance in yeast</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>Sphingolipid synthesis is required for heat stress resistance in yeast.</strong><br>
<em>Vanessa Cerantola, Isabelle Guillas, Carole Roubaty, Christine Vionnet, Dani√®le Uldry, Jens Knudsen and Andreas Conzelmann</em></p>
</blockquote>
<p>All mature Saccharomyces cerevisiae sphingolipids comprise inositolphosphorylceramides containing C26:0 or C24:0 fatty acids and either phytosphingosine or dihydrosphingosine. <span style="background-color: #f8bbd0"> Here we analysed the lipid profile of lag1D lac1D mutants lacking acyl-CoAdependent ceramide synthesis, which require the reverse ceramidase activity of overexpressed Ydc1p for sphingolipid biosynthesis and viability. These cells, termed 2D.YDC1, make sphingolipids containing exclusively dihydrosphingosine and an abnormally wide spectrum of fatty acids with between 18 and 26 carbon atoms.</span> Like wild-type cells, 2D.YDC1 cells stop growing when exposed to Aureobasidin A (AbA), an inhibitor of the inositolphosphorylceramide synthase AUR1, yet their ceramide levels remain very low. This finding argues against a current hypothesis saying that yeast cells do not require inositolphosphorylceramides and die in the presence of AbA only because ceramides build up to toxic concentrations. Moreover, W303lag1D lac1D ypc1D ydc1D cells, reported to be AbA resistant, stop growing on AbA after a certain number of cell divisions, most likely because AbA blocks the biosynthesis of anomalous inositolphosphorylsphingosides. Thus, data argue that inositolphosphorylceramides of yeast, the equivalent of mammalian sphingomyelins, are essential for growth. Data also clearly confirm that wild-type strains, when exposed to AbA, immediately stop growing because of ceramide intoxication, long before inositolphosphorylceramide levels become subcritical.</p>
<p><a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-2958.2009.06628.x">10.1111/j.1365-2958.2009.06628.x</a></p>
</details>
<hr>
</section>
</section>
<section id="guillas-et-al.-2001" class="level4">
<h4 class="anchored" data-anchor-id="guillas-et-al.-2001">Guillas et al.&nbsp;(2001)</h4>
<section id="sphingolipid-synthesis-is-required-for-heat-stress-resistance-in-yeast-1" class="level5">
<h5 class="anchored" data-anchor-id="sphingolipid-synthesis-is-required-for-heat-stress-resistance-in-yeast-1">Sphingolipid synthesis is required for heat stress resistance in yeast</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>C26-CoA-dependent ceramide synthesis of Saccharomyces cerevisiae is operated by Lag1p and Lac1p.</strong><br>
<em>Isabelle Guillas, Paul A.Kirchman, Rachel Chuard, Martine Pfefferli, James C.Jiang, S.Michal Jazwinski and Andreas Conzelmann</em></p>
</blockquote>
<p>Lag1p and Lac1p are two highly homologous membrane proteins of the endoplasmic reticulum (ER). When both genes are deleted, cells cannot transport glycosylphosphatidylinositol (GPI)-anchored proteins from the ER to the Golgi at a normal rate. Here we show that microsomes or detergent extracts from lag1Dlac1D double mutants lack an activity transferring C26 fatty acids from C26-coenzyme A onto dihydrosphingosine or phytosphingosine. As a consequence, in intact cells, the normal ceramides and inositolphosphorylceramides are drastically reduced. lag1Dlac1D cells compensate for the lack of normal sphingolipids by making increased amounts of C26 fatty acids, which become incorporated into glycerophospholipids. They also contain 20- to 25-fold more free long chain bases than wild type and accumulate very large amounts of abnormally polar ceramides. They make small amounts of abnormal mild baseresistant inositolphospholipids. The lipid remodelling of GPI-anchored proteins is severely compromised in lag1Dlac1D double mutants since only few and mostly abnormal ceramides are incorporated into the GPI anchors. synthesis may explain their role in determining longevity.</p>
<p><a href="http://emboj.embopress.org/cgi/doi/10.1093/emboj/20.11.2655">10.1093/emboj/20.11.2655</a></p>
</details>
<hr>
</section>
</section>
<section id="kol-et-al.-2025" class="level4">
<h4 class="anchored" data-anchor-id="kol-et-al.-2025">Kol et al.&nbsp;(2025)</h4>
<section id="optical-control-of-sphingolipid-biosynthesis-using-photoswitchable-sphingosines" class="level5">
<h5 class="anchored" data-anchor-id="optical-control-of-sphingolipid-biosynthesis-using-photoswitchable-sphingosines">Optical control of sphingolipid biosynthesis using photoswitchable sphingosines</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>Optical control of sphingolipid biosynthesis using photoswitchable sphingosines</strong><br>
<em>Matthijs Kol, Alexander J.E. Novak, Johannes Morstein, Christian Schr√∂er, Tolulope Sokoya, Svenja Mensing, Sergei M. Korneev, Dirk Trauner, Joost C.M. Holthuis</em></p>
</blockquote>
<p>Sphingolipid metabolism comprises a complex interconnected web of enzymes, metabolites and modes of regulation that influence a wide range of cellular and physiological processes. Deciphering the biological relevance of this network is challenging as numerous intermediates of sphingolipid metabolism are short-lived molecules with often opposing biological activities. Here, we introduce clickable, azobenzene-containing sphingosines, termed caSphs, as light-sensitive substrates for sphingolipid biosynthesis. Photo-isomerization of the azobenzene moiety enables reversible switching between a straight trans- and curved cis-form of the lipid‚Äôs hydrocarbon tail. Combining in vitro enzyme assays with metabolic labeling studies, we demonstrate that trans-to-cis isomerization of caSphs profoundly stimulates their metabolic conversion by ceramide synthases and downstream sphingomyelin synthases. These light-induced changes in sphingolipid production rates are acute, reversible, and can be implemented with great efficiency in living cells. Our findings establish caSphs as versatile tools with unprecedented opportunities to manipulate sphingolipid biosynthesis and function with the spatiotemporal precision of light.</p>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.10.24.619506v1">doi.org/10.1101/2024.10.24.619506</a></p>
</details>
<details>
<summary>
Notes
</summary>
<p>In this paper they worked with the previously created strain by A. Conzelmann (Cerantola et al.&nbsp;(2007)) but instead of using the murine Lass5, they engineered human CerS5 into the yeast.</p>
</details>
<hr>
</section>
</section>
<section id="ternes-et-al.-2009" class="level4">
<h4 class="anchored" data-anchor-id="ternes-et-al.-2009">Ternes et al.&nbsp;(2009)</h4>
<section id="sphingomyelin-synthase-sms2-displays-dual-activity-as-ceramide-phosphoethanolamine-synthase" class="level5">
<h5 class="anchored" data-anchor-id="sphingomyelin-synthase-sms2-displays-dual-activity-as-ceramide-phosphoethanolamine-synthase">Sphingomyelin synthase SMS2 displays dual activity as ceramide phosphoethanolamine synthase</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>Sphingomyelin synthase SMS2 displays dual activity as ceramide phosphoethanolamine synthase</strong><br>
<em>Isabelle Guillas, Paul A.Kirchman, Rachel Chuard, Martine Pfefferli, James C.Jiang, S.Michal Jazwinski and Andreas Conzelmann</em></p>
</blockquote>
<p>Sphingolipids are vital components of eukaryotic membranes involved in the regulation of cell growth, death, intracellular trafficking, and the barrier function of the plasma membrane (PM). While sphingomyelin (SM) is the major sphingolipid in mammals, previous studies indicate that mammalian cells also produce the SM analog ceramide phosphoethanolamine (CPE). Little is known about the biological role of CPE or the enzyme(s) responsible for CPE biosynthesis. SM production is mediated by the SM synthases SMS1 in the Golgi and SMS2 at the PM, while a closely related enzyme, SMSr, has an unknown biochemical function. We now demonstrate that SMS family members display striking differences in substrate specificity, with SMS1 and SMSr being monofunctional enzymes with SM and CPE synthase activity, respectively, and SMS2 acting as a bifunctional enzyme with both SM and CPE synthase activity. In agreement with the PM residency of SMS2, we show that both SM and CPE synthase activities are enhanced at the surface of SMS2-overexpressing HeLa cells. Our findings reveal an unexpected diversity in substrate specificity among SMS family members that should enable the design of specific inhibitors to target the biological role of each enzyme individually.</p>
<p><a href="https://linkinghub.elsevier.com/retrieve/pii/S0022227520306933">10.1194/jlr.M900230-JLR200</a></p>
</details>
<details>
<summary>
Notes
</summary>
<p>SMS1 was engierneerd into Œî4CerS5 strain (created by A. Conzelmann (Cerantola et al.&nbsp;(2007)) and led to sphingomyelin synthesis in this particular strain.</p>
</details>
<hr>
</section>
</section>
<section id="vionnet-et-al.-2011" class="level4">
<h4 class="anchored" data-anchor-id="vionnet-et-al.-2011">Vionnet et al.&nbsp;(2011)</h4>
<section id="yeast-cells-lacking-all-known-ceramide-synthases-continue-to-make-complex-sphingolipids-and-to-incorporate-ceramides-into-glycosylphosphatidylinositol-gpi-anchors" class="level5">
<h5 class="anchored" data-anchor-id="yeast-cells-lacking-all-known-ceramide-synthases-continue-to-make-complex-sphingolipids-and-to-incorporate-ceramides-into-glycosylphosphatidylinositol-gpi-anchors">Yeast Cells Lacking All Known Ceramide Synthases Continue to Make Complex Sphingolipids and to Incorporate Ceramides into Glycosylphosphatidylinositol (GPI) Anchors</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>Yeast Cells Lacking All Known Ceramide Synthases Continue to Make Complex Sphingolipids and to Incorporate Ceramides into Glycosylphosphatidylinositol (GPI) Anchors</strong><br>
<em>Christine Vionnet, Carole Roubaty, Christer S. Ejsing, Jens Knudsen, and Andreas Conzelmann</em></p>
</blockquote>
<p>In yeast, the inositolphosphorylceramides mostly contain C26:0 fatty acids. Inositolphosphorylceramides were considered to be important for viability because the inositolphosphorylceramide synthase AUR1 is essential. However, lcb1 cells, unable to make sphingoid bases and inositolphosphorylceramides, are viable if they harbor SLC1-1, a gain of function mutation in the 1-acyl-glycerol-3-phosphate acyltransferase SLC1. SLC1-1 allows the incorporation of C26:0 fatty acids into phosphatidylinositol (PI), thus generating PI , an abnormal, C26-containing PI, presumably acting as surrogate for inositolphosphorylceramide. Here we show that the lethality of the simultaneous deletion of the known ceramide synthases LAG1/LAC1/LIP1 and YPC1/YDC1 can be rescued by the expression of SLC1-1 or the overexpression of AUR1. Moreover, lag1 lac1 ypc1 ydc1 (4 ) quadruple mutants have been reported to be viable in certain genetic backgrounds but to still make some abnormal uncharacterized inositol-containing sphingolipids. Indeed, we find that 4 quadruple mutants make substantial amounts of unphysiological inositolphosphorylphytosphingosines but that they also still make small amounts of normal inositolphosphorylceramides. Moreover, 4 strains incorporate exogenously added sphingoid bases into inositolphosphorylceramides, indicating that these cells still possess an unknown pathway allowing the synthesis of ceramides. 4 cells also still add quite normal amounts of ceramides to glycosylphosphatidylinositol anchors. Synthesis of inositolphosphorylceramides and inositolphosphorylphytosphingosines is operated by Aur1p and is essential for growth of all 4 cells unless they contain SLC1-1. PI , however, is made without the help of Aur1p. Furthermore, mannosylation of PI is required for the survival of sphingolipid-deficient strains, which depend on SLC1-1. In contrast to lcb1 SLC1-1, 4 SLC1-1 cells grow at 37 ¬∞C but remain thermosensitive at 44 ¬∞C.</p>
<p><a href="https://linkinghub.elsevier.com/retrieve/pii/S0021925820520504">10.1074/jbc.M110.176875</a></p>
</details>
<hr>
</section>
</section>
<section id="cerantola-et-al.-2007-1" class="level4">
<h4 class="anchored" data-anchor-id="cerantola-et-al.-2007-1">Cerantola et al.&nbsp;(2007)</h4>
<section id="yeast-sphingolipids-do-not-need-to-contain-very-long-chain-fatty-acids" class="level5">
<h5 class="anchored" data-anchor-id="yeast-sphingolipids-do-not-need-to-contain-very-long-chain-fatty-acids">Yeast sphingolipids do not need to contain very long chain fatty acids</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>Yeast sphingolipids do not need to contain very long chain fatty acids</strong><br>
<em>Christine Vionnet, Carole Roubaty, Christer S. Ejsing, Jens Knudsen, and Andreas Conzelmann</em></p>
</blockquote>
<p>Synthesis of VLCFAs (very long chain fatty acids) and biosynthesis of DHS (dihydrosphingosine) both are of vital importance for Saccharomyces cerevisiae. The bulk of VLCFAs and DHS are used for ceramide synthesis by the Lag1p (longevity-assurance gene 1)/Lac1p (longevity-assurance gene cognate 1)/Lip1p (Lag1p/Lac1p interacting protein) ceramide synthase. LAG1 and LAC1 are redundant but LIP1 is essential. <span style="background-color: #f8bbd0"> Here we show that 4Œî (lag1Œîlac1Œîypc1Œîydc1Œî) cells devoid of all known endogenous ceramide synthesis pathways are unviable but can be rescued by the expression of Lass5, a mouse LAG1 homologue. Ceramide synthase activity of 4Œî.Lass5 cells only utilizes C16 and C18 fatty acids and does not require the help of Lip1p, an essential cofactor of Lag1p/Lac1p. </span> HPLC-electrospray ionization-MS/MS analysis demonstrated that in IPCs (inositolphosphorylceramides) of 4Œî.Lass5, the very long chain fatty acids (C26 and C24) account for &lt;1% instead of the normal &gt;97%. Notwithstanding, IPCs incorporated into glycosylphosphatidylinositol anchors of 4Œî.Lass5 show normal mobility on TLC and the ceramide- and raft-dependent traffic of Gas1p (glycophospholipid-anchored surface protein) from endoplasmic reticulum to Golgi remains almost normal. Moreover, the biosynthesis of C24:0 fatty acids remains essential. Thus, C24:0 and dihydrosphingosine are both necessary for survival of yeast cells even if they utilize C16 and C18 fatty acids for sphingolipid biosynthesis.</p>
<p><a href="10.1042/BJ20061128">10.1042/BJ20061128</a></p>
</details>
<hr>
</section>
</section>
<section id="menuz-et-al.-2009" class="level4">
<h4 class="anchored" data-anchor-id="menuz-et-al.-2009">Menuz et al.&nbsp;(2009)</h4>
<section id="protection-of-c.-elegans-from-anoxia-by-hyl-2-ceramide-synthase" class="level5">
<h5 class="anchored" data-anchor-id="protection-of-c.-elegans-from-anoxia-by-hyl-2-ceramide-synthase">Protection of <em>C. elegans</em> from anoxia by HYL-2 ceramide synthase</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>Protection of C. elegans from anoxia by HYL-2 ceramide synthase</strong><br>
<em>Vincent Menuz, Kate S Howell, S√©bastien Gentina, Sharon Epstein, Isabelle Riezman, Monique Fornallaz-Mulhauser, Michael O Hengartner, Marie Gomez, Howard Riezman, Jean-Claude Martinou</em></p>
</blockquote>
<p>Oxygen deprivation is rapidly deleterious for most organisms. However, Caenorhabditis elegans has developed the ability to survive anoxia for at least 48 hours. Mutations in the DAF-2/DAF-16 insulin-like signaling pathway promote such survival. We describe a pathway involving the HYL-2 ceramide synthase that acts independently of DAF-2. Loss of the ceramide synthase gene hyl-2 results in increased sensitivity of C. elegans to anoxia. C. elegans has two ceramide synthases, hyl-1 and hyl-2, that participate in ceramide biogenesis and affect its ability to survive anoxic conditions. In contrast to hyl-2(lf) mutants, hyl-1(lf) mutants are more resistant to anoxia than normal animals. HYL-1 and HYL-2 have complementary specificities for fatty acyl chains. These data indicate that specific ceramides produced by HYL-2 confer resistance to anoxia.</p>
<p><a href="10.1126/science.1168532">10.1126/science.1168532</a></p>
</details>
<hr>
</section>
</section>
</details></section>
<section id="sphingomyelin-synthase-expression-in-yeast" class="level2">
<h2 class="anchored" data-anchor-id="sphingomyelin-synthase-expression-in-yeast">üî¥Sphingomyelin Synthase Expression in Yeast</h2>
<blockquote class="blockquote">
<p>These studies describe the expression of SM synthase (from human and mouse origins) in <em>Saccharomyces cerevisiae</em>.</p>
</blockquote>
<details>
<summary>
<strong>Show Publications</strong>
</summary>
<section id="ternes-et-al.-2009-1" class="level4">
<h4 class="anchored" data-anchor-id="ternes-et-al.-2009-1">Ternes et al.&nbsp;(2009)</h4>
<section id="sphingomyelin-synthase-sms2-displays-dual-activity-as-ceramide-phosphoethanolamine-synthase-1" class="level5">
<h5 class="anchored" data-anchor-id="sphingomyelin-synthase-sms2-displays-dual-activity-as-ceramide-phosphoethanolamine-synthase-1">Sphingomyelin synthase SMS2 displays dual activity as ceramide phosphoethanolamine synthase</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>Sphingomyelin synthase SMS2 displays dual activity as ceramide phosphoethanolamine synthase</strong><br>
<em>Isabelle Guillas, Paul A.Kirchman, Rachel Chuard, Martine Pfefferli, James C.Jiang, S.Michal Jazwinski and Andreas Conzelmann</em></p>
</blockquote>
<p>Sphingolipids are vital components of eukaryotic membranes involved in the regulation of cell growth, death, intracellular trafficking, and the barrier function of the plasma membrane (PM). While sphingomyelin (SM) is the major sphingolipid in mammals, previous studies indicate that mammalian cells also produce the SM analog ceramide phosphoethanolamine (CPE). Little is known about the biological role of CPE or the enzyme(s) responsible for CPE biosynthesis. SM production is mediated by the SM synthases SMS1 in the Golgi and SMS2 at the PM, while a closely related enzyme, SMSr, has an unknown biochemical function. We now demonstrate that SMS family members display striking differences in substrate specificity, with SMS1 and SMSr being monofunctional enzymes with SM and CPE synthase activity, respectively, and SMS2 acting as a bifunctional enzyme with both SM and CPE synthase activity. In agreement with the PM residency of SMS2, we show that both SM and CPE synthase activities are enhanced at the surface of SMS2-overexpressing HeLa cells. Our findings reveal an unexpected diversity in substrate specificity among SMS family members that should enable the design of specific inhibitors to target the biological role of each enzyme individually.</p>
<p><a href="https://linkinghub.elsevier.com/retrieve/pii/S0022227520306933">10.1194/jlr.M900230-JLR200</a></p>
</details>
<details>
<summary>
Notes
</summary>
<p>SMS1 (among other SMS) was engierneerd into Œî4CerS5 strain (created by A. Conzelmann (Cerantola et al.&nbsp;(2007)) and led to sphingomyelin synthesis in this particular strain.</p>
</details>
<hr>
</section>
</section>
<section id="yang-et-al.-2007" class="level4">
<h4 class="anchored" data-anchor-id="yang-et-al.-2007">Yang et al.&nbsp;(2007)</h4>
<section id="identification-of-mouse-sphingomyelin-synthase-1-as-a-suppressor-of-bax-mediated-cell-death-in-yeast" class="level5">
<h5 class="anchored" data-anchor-id="identification-of-mouse-sphingomyelin-synthase-1-as-a-suppressor-of-bax-mediated-cell-death-in-yeast">Identification of mouse sphingomyelin synthase 1 as a suppressor of Bax-mediated cell death in yeast</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>Identification of mouse sphingomyelin synthase 1 as a suppressor of Bax-mediated cell death in yeast</strong><br>
<em>Zhao Yang , Chamel Khoury , Ga√´l Jean-Baptiste , Michael T. Greenwood</em></p>
</blockquote>
<p>We have identified mouse sphingomyelin synthase 1 as a novel suppressor of the growth inhibitory effect of heterologously expressed Bax. Yeast cells expressing sphingomyelin synthase 1 were also found to show an increased resistance to a variety of cytotoxic stimuli including hydrogen peroxide, osmotic stress and elevated temperature. Sphingomyelin synthase 1 functions by catalyzing the conversion of ceramide and phosphatidylcholine to sphingomyelin and diacylglycerol. Ceramide is an antiproliferative and proapoptotic sphingolipid whose level increases in response to a variety of stresses. Consistent with its biochemical function, yeast cells expressing sphingomyelin synthase 1 have an enhanced ability to grow in media containing the cell-permeable C2-ceramide analog as well as the ceramide precursor phytosphingosine. We also show that overexpression of AUR1, a potential yeast functional homolog of sphingomyelin synthase, also protects cells from osmotic stress. Taken together, these results suggest that sphingomyelin synthase 1 likely prevents cell death by counteracting stress-mediated accumulation of endogenous sphingolipids.</p>
<p><a href="https://doi.org/10.1111/j.1567-1364.2006.00052.x">10.1111/j.1567-1364.2006.00052.x</a></p>
</details>
<hr>
</section>
</section>
</details></section>
<section id="chemoenzymatic-ganglioside-synthesis" class="level2">
<h2 class="anchored" data-anchor-id="chemoenzymatic-ganglioside-synthesis">üü†Chemoenzymatic Ganglioside Synthesis</h2>
<blockquote class="blockquote">
<p>Jin et al.&nbsp;(2024) describes chemoenzymatic cascade assembly (MOCECA) strategy for the customized and large-scale synthesis of ganglioside analogs.</p>
<blockquote class="blockquote">
<p>The studies by Han et al.&nbsp;(2015) and Huang et al.&nbsp;(2015) are linked to this work through the enzyme sphingolipid ceramide N-deacylase (SCDase) and its applications in glycosphingolipid modification and synthesis.</p>
</blockquote>
</blockquote>
<details>
<summary>
<strong>Show Publications</strong>
</summary>
<section id="jin-et-al.-2024" class="level4">
<h4 class="anchored" data-anchor-id="jin-et-al.-2024">Jin et al.&nbsp;(2024)</h4>
<section id="a-modular-chemoenzymatic-cascade-strategy-for-the-structure-customized-assembly-of-ganglioside-analogs" class="level5">
<h5 class="anchored" data-anchor-id="a-modular-chemoenzymatic-cascade-strategy-for-the-structure-customized-assembly-of-ganglioside-analogs">A modular chemoenzymatic cascade strategy for the structure-customized assembly of ganglioside analogs</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>A modular chemoenzymatic cascade strategy for the structure-customized assembly of ganglioside analogs</strong><br>
<em>Xuefeng Jin, Hanchao Cheng, Xiaohui Chen, Xuefeng Cao, Cong Xiao, Fengling Ding, Huirong Qu, Peng George Wang, Yan Feng &amp; Guang-Yu Yang</em></p>
</blockquote>
<p>Gangliosides play vital biological regulatory roles and are associated with neurological system diseases, malignancies, and immune deficiencies. They have received extensive attention in developing targeted drugs and diagnostic markers. However, it is difficult to obtain enough structurally defined gangliosides and analogs especially at an industrial-relevant scale, which prevent exploring structure-activity relationships and identifying drug ingredients. Here, we report a highly modular chemoenzymatic cascade assembly (MOCECA) strategy for customized and large-scale synthesis of ganglioside analogs with various glycan and ceramide epitopes. We typically accessed five gangliosides with therapeutic promising and systematically prepared ten GM1 analogs with diverse ceramides. Through further process amplification, we achieved industrial production of ganglioside GM1 in the form of modular assembly at hectogram scale. Using MOCECA-synthesized GM1 analogs, we found unique ceramide modifications on GM1 could enhance the ability to promote neurite outgrowth. By comparing the structures with synthetic analogs, we further resolved the problem of contradicting descriptions for GM1 components in different pharmaceutical documents by reinterpreting the exact two-component structures of commercialized GM1 drugs. Because of its applicability and stability, the MOCECA strategy can be extended to prepare other glycosphingolipid structures, which may pave the way for developing new glycolipid drugs.</p>
<p><a href="https://doi.org/10.1038/s42004-024-01102-9">10.1038/s42004-024-01102-9</a></p>
</details>
<hr>
</section>
</section>
<section id="han-et-al.-2015" class="level4">
<h4 class="anchored" data-anchor-id="han-et-al.-2015">Han et al.&nbsp;(2015)</h4>
<section id="comprehensive-characterization-of-sphingolipid-ceramide-n-deacylase-for-the-synthesis-and-fatty-acid-remodeling-of-glycosphingolipids" class="level5">
<h5 class="anchored" data-anchor-id="comprehensive-characterization-of-sphingolipid-ceramide-n-deacylase-for-the-synthesis-and-fatty-acid-remodeling-of-glycosphingolipids">Comprehensive characterization of sphingolipid ceramide N-deacylase for the synthesis and fatty acid remodeling of glycosphingolipids</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>Comprehensive characterization of sphingolipid ceramide N-deacylase for the synthesis and fatty acid remodeling of glycosphingolipids</strong><br>
<em>Yun-Bin Han, Lie Wu, Jamie R. Rich, Feng-Tao Huang, Stephen G. Withers, Yan Feng &amp; Guang-Yu Yang</em></p>
</blockquote>
<p>Sphingolipid ceramide N-deacylase (SCDase) catalyzes reversible reactions in which the amide linkage in glycosphingolipids is hydrolyzed or synthesized. While SCDases show great value for the enzymatic synthesis of glycosphingolipids, they are relatively poorly characterized enzymes. In this work, the enzymatic properties of SCDase from Shewanella alga G8 (SA_SCD) were systematically characterized and compared with the commercially available SCDase from Pseudomonas sp. TK4 (PS_SCD). The optimal pH values for the hydrolytic and synthetic activity of SA_SCD were pH 6.0 and pH 7.5, respectively. Both activities were strongly inhibited by Zn2+ and Cu2+, while Fe2+, Co2+, Ni2+, Mn2+, Ca2+, and Mg2+ promoted the hydrolytic activity but inhibited the synthetic activity. SA_SCD showed very broad substrate specificity both in hydrolysis and synthesis. Importantly, SA_SCD has a broader specificity for acyl donor acceptance than does PS_SCD, especially for unsaturated fatty acids and fatty acids with very short or long acyl chains. Further kinetic analysis revealed that the k cat/K M value for the hydrolytic activity of SA_SCD was 8.9-fold higher than that of PS_SCD for GM1a, while the values for the synthetic activity were 38-fold higher for stearic acid and 23-fold higher for lyso-GM1a (d18:1) than those of PS_SCD, respectively. The broad fatty acid specificity and high catalytic efficiency, together with the ease of expression of SA_SCD in Escherichia coli, make it a better biocatalyst than is PS_SCD for the synthesis and structural remodeling of glycosphingolipids.</p>
<p><a href="10.1007/s00253-015-6421-8">10.1007/s00253-015-6421-8</a></p>
</details>
<hr>
</section>
</section>
<section id="huang-et-al.-2015" class="level4">
<h4 class="anchored" data-anchor-id="huang-et-al.-2015">Huang et al.&nbsp;(2015)</h4>
<section id="a-facile-method-for-controlling-the-reaction-equilibrium-of-sphingolipid-ceramide-n-deacylase-for-lyso-glycosphingolipid-production" class="level5">
<h5 class="anchored" data-anchor-id="a-facile-method-for-controlling-the-reaction-equilibrium-of-sphingolipid-ceramide-n-deacylase-for-lyso-glycosphingolipid-production">A facile method for controlling the reaction equilibrium of sphingolipid ceramide N-deacylase for lyso-glycosphingolipid production</h5>
<details>
<summary>
Abstract
</summary>
<blockquote class="blockquote">
<p><strong>A facile method for controlling the reaction equilibrium of sphingolipid ceramide N-deacylase for lyso-glycosphingolipid production</strong><br>
<em>Feng-Tao Huang, Yun-Bin Han, Yan Feng and Guang-Yu Yang</em></p>
</blockquote>
<p>Lyso-glycosphingolipids (lyso-GSLs), the N-deacylated forms of glycosphingolipids (GSLs), are important synthetic intermediates for the preparation of GSL analogues. While lyso-GSLs can be produced by hydrolyzing natural GSLs using sphingolipid ceramide N-deacylase (SCDase), the yield for this reaction is usually low because SCDase also catalyzes the reverse reaction, ultimately establishing an equilibrium between hydrolysis and synthesis. In the present study, we developed an efficient method for controlling the reaction equilibrium by introducing divalent metal cation and detergent in the enzymatic reaction system. In the presence of both Ca2+ and taurodeoxycholate hydrate (TDC), the generated fatty acids were precipitated by the formation of insoluble stearate salts, and pushing the reaction equilibrium toward hydrolysis. The yield of GM1 hydrolysis can be achieved as high as 96%, with an improvement up to 45% compared with non-optimized condition. In preparative scale, 75 mg of lyso-GM1 was obtained from 100 mg of GM1 with a 90% yield, which is the highest reported yield to date. The method can also be used for the efficient hydrolysis of a variety of GSLs and sphingomyelin, thus it should serve as a facile, easily scalable, and general tool for lyso-GSL production to facilitate further GSL researches.</p>
</details>
</section>
</section>
</details></section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "Óßã";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>